tests/test_cache_lock_file_perf.py

```diff
@@ -0,0 +1,110 @@
+import multiprocessing
+import threading
+import time
+from msal_extensions import FilePersistence, CrossPlatLock
+
+
+def validate_thread_locking_file(num):
+    with open("C:/Users/abpati/Desktop/test/lockfilelog.txt") as handle:
+        data = handle.read()
+    time_list = []
+    for line in data.split("\n"):
+        start, end = line.split("-")
+        time_list.append([start, end])
+    if len(time_list) != num:
+        print("Threads count dont match ")
+        return
+    time_list.sort()
+    prev = None
+    for interval in time_list:
+        start, end = interval
+        if start > end:
+            print("Intervals start bigger than end")
+            return
+        if prev:
+            if start < prev:
+                print("overlapping")
+        prev = start
+    print("success")
+    return
+
+
+def validate_result_in_cache(num):
+    with open("C:/Users/abpati/Desktop/test/msal.cache") as handle:
```

Comment:
Next step is to not hard code any specific path, so that this test case would be able to be run at anywhere, including CI/CD.